### PR TITLE
Starlay Finance: add aUSD

### PR DESCRIPTION
**Pull request by qudacki**
add [$aUSD](https://www.coingecko.com/en/coins/acala-dollar)

### CODE REVIEW

**Comment by llamatester** [comment]
Error while running adapter at projects/starlay: 
        
 
  
    
    
    Error: invalid BigNumber value (argument="value", value="62001078.132701136395476854", code=INVALID_ARGUMENT, version=bignumber/5.6.2)
    Truncated error stack:
        at Function.BigNumber.from (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@ethersproject/bignumber/lib/bignumber.js:239:23)
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:331:16
        at Array.forEach (<anonymous>)
        at fixBalances (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:325:28)
        at computeTVL (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:337:3)
        at getTvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:60:30)

**Comment by llamatester** [comment]
The adapter at projects/starlay exports TVL: 
        
 
 
    astar-staking             333.84 k
    astar                     9.40 M
    astar-borrowed            18.63 M
    staking                   333.84 k
    borrowed                  18.63 M
    
    total                    9.40 M

File: projects/starlay/starlay.js [hunk 1]
```diff
@@ -83,6 +83,7 @@ async function getStarlayTvl(
     block,
     chain,
   });
+  console.log(balanceOfUnderlying)
```
Comment (g1nt0ki) [inline projects/starlay/starlay.js:86]: PR is good, remove this line

**Comment by llamatester** [comment]
The adapter at projects/starlay exports TVL: 
        
 
 
    astar-staking             329.22 k
    astar                     9.37 M
    astar-borrowed            18.63 M
    staking                   329.22 k
    borrowed                  18.63 M
    
    total                    9.37 M

File: projects/starlay/starlay.js (hunk 1 above)
Comment (qudacki) [reply projects/starlay/starlay.js]: Thanks! Fixed.

**Pull request merged**